finisher_api.go

```diff
@@ -354,6 +354,8 @@ func (db *DB) FirstOrCreate(dest interface{}, conds ...interface{}) (tx *DB) {
 		} else {
 			tx.Error = result.Error
```

Comment:
Seems this part is useless

Reply:
Thank you for your review :bow:

changed with this commit.
[delete useless part](https://github.com/go-gorm/gorm/pull/5367/commits/231b98ae7afeaedb5235a4753e47f66dd8c5e04b)